client/my-sites/plans-grid/components/comparison-grid/index.tsx

```diff
@@ -872,7 +873,10 @@ const FeatureGroup = ( {
 		firstSetOfFeatures,
 	] );
 	const features = featureGroup.get2023PricingGridSignupWpcomFeatures();
-	const featureObjects = getPlanFeaturesObject( allFeaturesList, features );
+	const featureObjects = filterUnusedFeaturesObject(
```

Comment:
Hey, @chriskmnds 👋 

Appreciate if you could help me check if I'm doing anything that goes against the direction you'd like plan grid to go since I'm aware of a refactor around it. I intend to hide features where none of the plans are using from the grid.

Reply:
Ah - sorry for the wait, and thanks for waiting. I left a comment below and can follow up a little better next :-)